repose-aggregator/tests/release-verification/src/docker/deb/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# To build this image, use a command like the following:
+# docker build --build-arg REPOSE_VERSION=latest -t repose-container .
+# To run the built image, use a command like the following:
+# docker run --rm repose-container
+FROM ubuntu:xenial
+
+MAINTAINER The Repose Team <reposecore@rackspace.com>
+
+# This build-arg is used to pass the Repose version number which will be set up in this image.
+ARG REPOSE_VERSION
+
+# The service ports -- one for accepting HTTP traffic to Repose, and the other for connecting a JDWP debugger
+# Note that these are not exposed to the host by default, but can be using the Docker CLI -p or -P flag.
+EXPOSE 8080 10037
+
+RUN apt-get update -qq && apt-get install -y -qq \
+    wget \
+    curl \
+    python-pip
+RUN wget --quiet http://mirrors.kernel.org/ubuntu/pool/universe/r/rlwrap/rlwrap_0.41-1build1_amd64.deb
+RUN dpkg -i rlwrap_0.41-1build1_amd64.deb
+RUN wget --quiet https://deb.nodesource.com/node_0.12/pool/main/n/nodejs/nodejs_0.12.7-1nodesource1~trusty1_amd64.deb
+RUN dpkg -i nodejs_0.12.7-1nodesource1~trusty1_amd64.deb
```

Comment:
NodeJS should be installed by Bill's new script.

Do the same for the RPM Dockerfile.